packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -361,180 +335,99 @@ flutter:
       Platform: () => platform,
       ProcessManager: () => FakeProcessManager.any(),
     });
+  });
 
-    group('flavors feature', () {
-      Future<ManifestAssetBundle> buildBundleWithFlavor(String? flavor) async {
-        final ManifestAssetBundle bundle = ManifestAssetBundle(
-          logger: logger,
-          fileSystem: testFileSystem,
-          platform: platform,
-          flutterRoot: Cache.defaultFlutterRoot(
-            platform: platform,
-            fileSystem: testFileSystem,
-            userMessages: UserMessages(),
-          ),
-          splitDeferredAssets: true,
-        );
-
-        await bundle.build(
-          packagesPath: '.packages',
-          flutterProject: FlutterProject.fromDirectoryTest(testFileSystem.currentDirectory),
-          flavor: flavor,
-        );
-        return bundle;
-      }
+  group('AssetBundle.build (hermetic)', () {
```

Comment:
why hermetic in the group name? are some of these not hermetic?

Reply:
Yes, see the group above named "AssetBundle.build"

Reply:
Alternatively, I can rename the other group to `AssetBundle.build (non-hermetic)`

Reply:
Basically the point of this was I didn't want to reuse the polluted scope of the existing group